src/components/message/message.style.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable max-len */
```

Comment:
Is this needed?

Reply:
it's for 13 line (I kept getting maximum line length error)

Reply:
You can use the following just before the offending line:

`/* eslint-disable-next-line max-len */`

Which should just disable the rule on _that_ line.